spark-driver/src/main/resources/spark.properties

```diff
@@ -5,7 +5,7 @@ spark.master=local[2]
 
 spark.application-name=Spark Presentation
 
-spark.distributed-libraries=file:///Users/tmatyashovsky/Workspace/spark-samples-jeeconf-kyiv/spark-distributed-library/target/spark-distributed-library-1.0-SNAPSHOT-uber.jar
+spark.distributed-libraries=file://${app.home:.}/attendee-aggregator/spark-distributed-library/target/spark-distributed-library-1.0-SNAPSHOT-uber.jar
```

Comment:
Do we really need to have some "home" folder? Maybe it will be easier to work in the same folder where app is run?